blosc/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
 # a simple way to detect that we are using CMAKE
 add_definitions(-DUSING_CMAKE)
 
-set(INTERNAL_LIBS ${CMAKE_SOURCE_DIR}/internal-complibs)
+set(INTERNAL_LIBS ${PROJECT_SOURCE_DIR}/internal-complibs)
 
 # Hide symbols by default unless they're specifically exported.
 # This makes it easier to keep the set of exported symbols the
 # same across all compilers/platforms.
-set(CMAKE_C_VISIBILITY_PRESET hidden)
+#set(CMAKE_C_VISIBILITY_PRESET hidden)
```

Comment:
Ah, another stuff-up on my part. According to a CMake warning, this should only be enabled for the shared lib, and I was going to work out how to set this in a target specific way, but obviously forgot. I can either re-enable it if that warning is irrelvant, or work out how to make it SO-specific